### DISCUSSION TITLE
Fieldset

### DISCUSSION THREAD

**Discussion by mgkz** [tobimori/kirby-dreamform#43] [Feature Requests]
Create fieldsets with legends to group inputs

**Comment by sachbearbeiter** [comment]
And I miss simple intro, legend or help text fields. Very simple without any processing, but incredibly important in forms that are not just simple contact forms. Also dividing lines (or attributes to which they can be attached).

**Reply by tobimori** [reply]
See https://github.com/tobimori/kirby-dreamform/discussions/9
